app/com/arpnetworking/kairos/service/KairosDbServiceImpl.java

```diff
@@ -219,7 +221,7 @@
         return response;
     }
 
-    private static MetricsQuery useAvailableRollups(
+    /* package private */ static MetricsQuery useAvailableRollups(
```

Comment:
package private for tests?

Reply:
Correct! (I'm open to other ways of making it unit-testable too, if you have any you'd prefer.)